robot_msgs/PointStamped point_stamped
---
